### ISSUE TITLE
Missing close or choose button in file picker dialog after update to 0.7.5

### ISSUE THREAD

**Issue by laurentiu2** [nextcloud/mail#630]
Same thing as #626 but with the current stable version 12.0.3, only after update to 0.7.5
![screen shot 2017-11-27 at 8 25 27 pm](https://user-images.githubusercontent.com/29857699/33282660-fb31cabc-d3b1-11e7-8f4f-6dfc6e08b142.png)

**Comment by ChristophWurst** [comment]
No need to double-post. Let's continue in https://github.com/nextcloud/mail/issues/626#issuecomment-347277437 😉

**Comment by ChristophWurst** [comment]
> only after update to 0.7.5

But still, good to know. That means it was broken recently.

**Comment by laurentiu2** [comment]
RIght 😋 
Still no luck for me to save attachments directly to NC..

**Comment by ChristophWurst** [comment]
> Still no luck for me to save attachments directly to NC..

File a new ticket for this. Thanks!
